Electronics/PCB/AstroPlant_ExtensionBoard/astroplant-extension-board-mk0-6-1/.gitignore

```diff
@@ -0,0 +1,9 @@
+# KiCAD project
+
+*.pro
```

Comment:
I would only ignore `.bak` and `.lck` files, because they are clearly work files of the program.
It is useful to see those other files and if you gitignore them they will not be shown when committing.

Reply:
I left : 
*.bak
*.lck
*autosave*
*sch-bak
The last two appear when the project is opened

Reply:
Cool thanks